app-modules/form/resources/views/infolists/components/signature-entry.blade.php

```diff
@@ -0,0 +1,15 @@
+<x-dynamic-component
+    :component="$getEntryWrapperView()"
+    :entry="$entry"
+>
+    @php
+        $state = $getState();
+    @endphp
+
+    @if (filled($state))
+        <img
```

Comment:
Can we add a background in dark mode to increase contrast?

<img width="355" alt="Screenshot 2023-10-25 at 10 26 47 PM" src="https://github.com/canyongbs/assistbycanyongbs/assets/10821263/c1d1c51f-4abd-46e5-9110-cf6ab5ef28dd">

Reply:
Done!